buildSrc/src/main/kotlin/java-library-conventions.gradle.kts

```diff
@@ -180,6 +183,29 @@ tasks.jar {
 				"Implementation-Vendor" to "junit.org"
 		)
 	}
+
+	withConvention(BundleTaskConvention::class) {
+		setBnd("""
+			Bundle-Name: ${project.description}
```

Comment:
Isn't this evaluated before the descriptions are set?

Reply:
I split this up so that only that one pieces is during `afterEvaluate`